ipaserver/install/dsinstance.py

```diff
@@ -265,12 +261,14 @@ def __common_setup(self, enable_ssl=False):
         self.step("creating directory server instance", self.__create_instance)
         self.step("updating configuration in dse.ldif", self.__update_dse_ldif)
         self.step("restarting directory server", self.__restart_instance)
+        self.step("enabling ldapi", self.__enable_ldapi)
```

Comment:
You're right, it is possible to connect to ldap right after the instance is created. I didn't notice the service is stopped before updating the dse. Thanks, fixed in next synchronization.